tests/report_tests/plugin_tests/system.py

```diff
@@ -0,0 +1,78 @@
+# This file is part of the sos project: https://github.com/sosreport/sos
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# version 2 of the GNU General Public License.
+#
+# See the LICENSE file in the source distribution for further information.
+
+import os
+from sos_tests import StageOneReportTest, StageTwoReportTest
+
+
+class SystemPluginTest(StageOneReportTest):
+    """Basic sanity check to make sure common config files are collected
+
+    :avocado: tags=stageone
+    """
+
+    sos_cmd = '-o system'
+
+    def test_system_files_collected(self):
+        self.assertFileGlobInArchive("/proc/sys")
+        if os.path.isdir("/etc/sysconfig"):
+            self.assertFileGlobInArchive("/etc/sysconfig")
+        self.assertFileGlobInArchive("/etc/default")
+        self.assertFileGlobInArchive("/etc/environment")
+
+    def test_system_files_forbidden(self):
+        self.assertFileGlobNotInArchive("/proc/sys/net/ipv4/route/flush")
+        self.assertFileGlobNotInArchive("/proc/sys/net/ipv6/route/flush")
+        self.assertFileGlobNotInArchive("/proc/sys/net/ipv6/neigh/" +
+                                        "*/retrans_time")
+        self.assertFileGlobNotInArchive("/proc/sys/net/ipv6/neigh/" +
+                                        "*/base_reachable_time")
+        self.assertFileGlobNotInArchive("/etc/default/grub.d/" +
+                                        "50-curtin-settings.cfg")
+
+    def test_system_cmd_collected(self):
+        self.assertFileGlobInArchive("ld.so_--help")
+        self.assertFileGlobInArchive("ld.so_--list-diagnostics")
+        self.assertFileGlobInArchive("ld.so_--list-tunables")
+
+
+class SystemScrubTest(StageTwoReportTest):
+    """Ensure that environment, default and sysconfig are picked up
+    and properly scrubbed
+
+    :avocado: tags=stagetwo,scrub
+    """
+    sos_cmd = '-o system'
+    files = [
+        ('../../../tests/test_data/system_test_data', '/etc/environment'),
+        ('../../../tests/test_data/system_test_data', '/etc/default/proxy'),
+        ('../../../tests/test_data/system_test_data', '/etc/default/proxy1'),
```

Comment:
Ubuntu tests are failing here:
```
 (070/116) tests/report_tests/plugin_tests/system.py:SystemScrubTest.test_archive_created: STARTED
 (070/116) tests/report_tests/plugin_tests/system.py:SystemScrubTest.test_archive_created:  ERROR: [Errno 2] No such file or directory: '/etc/sysconfig/proxy1' (0.02 s)
 (071/116) tests/report_tests/plugin_tests/systemd.py:SystemdScrubTest.test_systemd_files_collected: STARTED
```

Do we really need the 1 at the end of this file and /etc/sysconfig/proxy1 or is that a typo?

Reply:
Issue is not with naming like proxy or proxy1, the scrub in code should obfuscate any file in /etc/sysconfig. 

Issue seems to be that ubuntu do not have /etc/sysconfig at all, due to that i have made separated tests for all + specific /etc/sysconfig for redhat + centos distribution. As you can see in last pipeline results of all stagetwo tests passed.